controllers/product.controller.js

```diff
@@ -0,0 +1,32 @@
+export let productController = {
```

Comment:
There's no need to create a variable "let productController" here.
A best practice is to include exporting right at the end of your file like this:
`export { ... };`
Or if you still want to create a default export, create a const variable before exporting it like so:
`export default yourVariable;`

The choice depends on the way you would like to import it further in the app.
E.g. 
![image](https://user-images.githubusercontent.com/25229823/62464471-5e343280-b795-11e9-9545-3a7957dbb595.png)